checks/core.py

```diff
@@ -122,3 +122,9 @@ def checkOperatingSystem(lines):
             return "windows"
         elif 'linux' in s:
             return "linux"
+
+
+def checkPortableMode(lines):
+    if search('Portable mode: true', lines):
+        return [LEVEL_INFO, "Portable Mode",
+                """You are running OBS in portable mode. This means that OBS will store it's settings in the same directory as the executable. This is useful if you want to run OBS from a flash drive or other removable media."""]
```

Comment:
Technically, the settings are not stored in the same directory as the executable. It might be sufficient to say "store its settings with the executable" as attempts to be more technically correct (e.g., "adjacent to" or "near") are probably just confusing.

Reply:
A reply to this additional feedback would be appreciated.

Reply:
Yes, I made this change but my reply got lost in the file permission chaos I created.